consoles/sshVirtsh.pm

```diff
@@ -496,6 +498,22 @@ sub resume ($self) {
 
 sub get_remote_vmm ($self) { $bmwqemu::vars{VMWARE_REMOTE_VMM} // '' }
 
+sub _encode_config ($self, $config, $key) {
+    unless ($config) {
+        croak "Please provide a configuration file to encode, variable $key has to be set!";
```

Comment:
If this isn't easily testable in the unit test you need to mark it as uncoverable
```suggestion
        croak "Please provide a configuration file to encode, variable $key has to be set!"; # uncoverable statement
```